webapp/i18n/en.json

```diff
@@ -1,51 +1,165 @@
 {
   "/WMCDd": "Something went wrong!",
+  "0Ihpmc": "Record call",
+  "0LuKMT": "You're already in a call",
+  "0cE6s2": "Calls can't be initiated in an insecure context",
+  "0fFRUQ": "Start presenting",
   "0hdUh1": "An internal error occurred preventing you to join the call. Please try again.",
   "0kSG/+": "Notify Admin",
+  "1QvHUW": "Your recording will end in {count, plural, =1 {# minute} other {# minutes}}.",
+  "22Lra1": "Here's the call recording",
+  "22uezg": "You are unmuted. Click <muteIcon/> to mute.",
+  "2T4EGD": "You're already in a call with {participant}.",
   "2n0xcg": "Enable calls",
   "3wS4fn": "Upgrade to use calls in Channels",
   "47FYwb": "Cancel",
+  "5+bTjM": "Mattermost Cloud Professional feature",
+  "5ClUrE": "Something went wrong with the recording",
+  "5DNd2p": "Recording has stopped. Processing...",
+  "5LvABK": "others",
```

Comment:
```suggestion
  "5LvABK": "Others",
```

Reply:
I think this is supposed to be as is, just missing some context as it ultimately goes into this kind of sentence:

`UserA, UserB & N others`, where N is the number of additional participants with their hand raised.